packages/debug/src/browser/debug-prefix-configuration.ts

```diff
@@ -119,4 +135,16 @@ export class DebugPrefixConfiguration implements CommandContribution, CommandHan
         this.debugConfigurationManager.current = { ...configuration };
         this.commandRegistry.executeCommand(DebugCommands.START.id);
     }
+
+    protected updateStatusBar(): void {
+        const text: string = this.debugConfigurationManager.current
+            ? this.debugConfigurationManager.current.configuration.name
+            : 'Start Debugging';
```

Comment:
in VS Code there is empty text if there are no configurations, try to delete launch.json